classes/shortcodes.php

```diff
@@ -181,7 +210,9 @@ function do_shortcode( $args ) {
 				}
 
 				// Determine location IDs
-				$location_ids = empty( $args['location_ids'] ) ? array( $post->ID ) : explode( ',', $args['location_ids'] );
+				$location_ids = empty( $args['location_ids'] )
+					? array( $post->ID )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->